geoips/interfaces/module_based/output_checkers.py

```diff
@@ -164,7 +164,7 @@ def write_remove_temp_files_to_file(remove_temp_files, diffdir):
             "temp files from gunzipped files scratch path.",
             len(remove_temp_files),
         )
-        LOG.interactive(f"  source {fname_rmtemp}")
+        print(f"  source {fname_rmtemp}")
```

Comment:
Why switch from logging to printing here?

Reply:
So I can triple click on the source line and not get the logging information, so I can just paste it into the terminal...

Reply:
Do we need to put these back as logging, or is it ok as print?

Reply:
Added back! Thanks mindy :)